### PR TITLE
feat: fare media vote passed update

### LINKED ISSUE #114: Update website with Fare media
As a result of https://github.com/google/transit/pull/355

I believe [Reference.md](https://github.com/MobilityData/gtfs.org/blob/main/docs/schedule/reference.md) will be updated automatically, but we need to manually update:
- the revision history ([CHANGES.md](https://github.com/google/transit/blob/master/gtfs/CHANGES.md) on google/transit, [process.md](https://github.com/MobilityData/gtfs.org/blob/main/docs/schedule/process.md) on ours)
- this page: https://github.com/MobilityData/gtfs.org/blob/main/docs/schedule/changes/index.md

### PR DESCRIPTION
**Pull request by emmambd**
Announcing that the fare media vote has passed on gtfs.org. Includes requested changes for #114, including revision history and changes overview.

To add: 

- [x] - Translations of changes to reference.MD
- [x] - Updates to History section in Fares v2 page
- [x] - Translations for adoption tracker text in Fares v2 page

### CODE REVIEW

**Review by isabelle-dr — APPROVED** [review]
LGTM

File: docs/extensions/fares-v2.ja.md
Comment (josee-sabourin) [inline docs/extensions/fares-v2.ja.md:1]: generally seems to be markdown errors in the history section, ex: year only in bold and not month

File: docs/schedule/reference.de.md [hunk 1]
```diff
@@ -349,20 +351,35 @@ Beispiele für die Angabe einer Tarifstruktur mit [fare_rules.txt](#farerulestxt
...
+Datei: **Fakultativ**
+
+Primary Key (`fare_media_id`)
+
+Beschreibung der verschiedenen Tarifmedien, die für die Nutzung von Tarifprodukten verwendet werden können. Tarifmedien sind physische oder virtuelle Träger, die für die Darstellung und/oder Validierung eines Tarifprodukts verwendet werden.
+
+|  Feldname   | Typ | Anwesenheit | Beschreibung  |
+|  ------ | ------ | ------ | ------ |
+|  `fare_media_id` | Eindeutige ID  | **Erforderlich** | Identifiziert ein Fahrgeldmedium.|
+|  `fare_media_name` | Text | Fakultativ | Name des Tarifmediums.<br/><br/>Für Tarifmedien, bei denen es sich um Transitkarten`(fare_media_type =2`) oder mobile Apps`(fare_media_type =4`) handelt, sollte der `fare_media_name` angegeben werden und mit dem Namen übereinstimmen, den die Organisationen, die sie bereitstellen, für die Fahrgäste verwenden.<br/><br/>|
```
Comment (josee-sabourin) [inline docs/schedule/reference.de.md]: opening brackets formatted as code on both occasions

File: docs/schedule/reference.de.md [hunk 2]
```diff
@@ -349,20 +351,35 @@ Beispiele für die Angabe einer Tarifstruktur mit [fare_rules.txt](#farerulestxt
...
+|  `fare_media_type` | Enum | **Erforderlich** | Der Typ der Tarifmedien. Gültige Optionen sind:<br/><br/>0 - Keine. Wird verwendet, wenn beim Kauf oder bei der Validierung eines Fahrpreisprodukts keine Fahrpreismedien beteiligt sind, z. B. bei Barzahlung an einen Fahrer oder Schaffner ohne physischen Fahrschein.<br/>2 - Physische Transitkarte, auf der Fahrscheine, Pässe oder ein Geldwert gespeichert sind.<br/>3 - cEMV (kontaktlose Europay, Mastercard und Visa) als Open-Loop-Token-Container für kontobasiertes Ticketing.<br/>4 - Mobile App, auf der virtuelle Transitkarten, Fahrscheine, Pässe oder ein Geldwert gespeichert sind.<br/><br/><br/><br/><br/>|
```
Comment (josee-sabourin) [inline docs/schedule/reference.de.md]: enums 
(numbers) not formatted as code

File: docs/schedule/reference.es.md [hunk 3]
```diff
@@ -117,6 +118,7 @@ Esta especificación define los siguientes archivos:
...
 | [fare_rules.txt](#fare_rulestxt)                   | Opcional    | Normas de aplicación de las tarifas para los itinerarios.                                                                                                                                                                                                                                                                           |
+| [fare_media.txt](#fare_mediatxt) | Opcional | Describir los medios tarifarios que pueden emplearse para utilizar productos tarifarios.<br/><br/>El archivo [fare_media](#fare_mediatxt).txt describe conceptos que no están representados en [fare_attributes.txt](#fare_attributestxt) xt y fare_rules [fare_rules.txt](#fare_rulestxt)txt. Como tal, el uso de [fare_media](#fare_mediatxt).txt es totalmente independiente de los archivos [fare_attributes.txt](#fare_attributestxt) xt y fare_rules. [fare_rules.txt](#fare_rulestxt). |
```
Comment (josee-sabourin) [inline docs/schedule/reference.es.md]: First sentence:
- .txt not included in hyperlink in first  fare_media.txt
- extra xt after fare_attributes.txt hyperlink
- extra fare_rules before and extra txt after fare_rules.txt

Second sentence:
- extra xt after fare_attributes.txt
- extra fare_rules before and extra txt after fare_rules.txt

File: docs/schedule/reference.es.md [hunk 4]
```diff
@@ -349,20 +351,35 @@ Para ver ejemplos que demuestran cómo especificar una estructura de tarifas con
...
+
+|  Nombre del campo | Tipo | Presencia | Descripción|
+|  ------ | ------ | ------ | ------ |
+|  `fare_media_id` | ID único  | **Exigida** | Identifica un soporte tarifario.|
+|  `fare_media_name` | Texto  | Opcional   | Name of the fare media.<br><br>For fare media which are transit cards (`fare_media_type =2`) or mobile apps (`fare_media_type =4`), the `fare_media_name` should be included and should match the rider-facing name used by the organizations delivering them. |
+|  `fare_media_type` | Enum | **Exigida** | El tipo de medio de tarifa. Las opciones válidas son:<br/><br/>0 - Ninguno. Se utiliza cuando no hay ningún medio de pago implicado en la compra o validación de un producto tarifario, como el pago en efectivo a un conductor o revisor sin billete físico.<br/>2 - Tarjeta de transporte física que ha almacenado billetes, pases o valor monetario.<br/>3 - cEMV (Europay, Mastercard y Visa sin contacto) como contenedor de tokens de bucle abierto para la emisión de billetes basada en cuentas.<br/>4 - Aplicación móvil que ha almacenado tarjetas de transporte virtuales, billetes, pases o valor monetario.<br/><br/><br/><br/><br/>|
```
Comment (josee-sabourin) [inline docs/schedule/reference.es.md]: enum numbers not formatted as code

File: docs/schedule/reference.fr.md [hunk 5]
```diff
@@ -117,6 +118,7 @@ Cette spécification définit les fichiers suivants :
...
 | [fare_rules.txt](#fare_rulestxt)                   | Optionnelle | Règles d'application des tarifs pour les itinéraires.<br/><br/>Conditionnellement obligatoire :<br/>- **Obligatoire** si [fare_attributes.txt](#fare_attributestxt) est défini.<br/>- **Interdit** autrement.                                                                                                                                                                                                                                                                                           |
+| [fare_media.txt](#fare_mediatxt) | Optionnelle | Décrire les supports tarifaires qui peuvent être utilisés pour utiliser les produits tarifaires.<br/><br/>Le fichier [fare_media.txt](#fare_mediatxt) décrit des concepts qui ne sont pas représentés dans les fichiers [fare_attributes.txt](#fare_attributestxt) xt et [fare_rules.txt](#fare_rulestxt). En tant que tel, l'utilisation du [fichier fare_media.txt](#fare_mediatxt) est entièrement distincte des fichiers [fare_attributes.txt](#fare_attributestxt) et [fare_rules.txt](#fare_rulestxt) |
```
Comment (josee-sabourin) [inline docs/schedule/reference.fr.md]: extra xt after fare_attributes.txt in first sentence

File: docs/schedule/reference.id.md [hunk 6]
```diff
@@ -117,6 +118,7 @@ Spesifikasi ini mendefinisikan file-file berikut:
...
 | [fare_rules.txt](#fare_rulestxt)                   | Opsional                      |  Aturan untuk menerapkan tarif untuk rencana perjalanan.                                                                                                                                                                                                                                                      |
+| [fare_media.txt](#fare_mediatxt) | Opsional | Mendeskripsikan media tarif yang dapat digunakan untuk menggunakan produk tarif.<br/><br/>File [fare_media.txt](#fare_mediatxt) menjelaskan konsep yang tidak terwakilifare_attributes.txt Danfare_rules.txt . Dengan demikian, penggunaan [fare_media.txt](#fare_mediatxt) sepenuhnya terpisah dari filefare_attributes.txt Danfare_rules.txt . |
```
Comment (josee-sabourin) [inline docs/schedule/reference.id.md]: missing hyperlinks for fare_attributes.txt and fare_rules.txt in both sentences

File: docs/schedule/reference.id.md [hunk 7]
```diff
@@ -353,20 +355,35 @@ Untuk contoh yang menunjukkan cara menentukan struktur tarif dengan fare_rules.t
...
+| `fare_media_id`   |  Identitas unik | ** Yg dibutuhkan** |  Mengidentifikasi media tarif.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```
Comment (josee-sabourin) [inline docs/schedule/reference.id.md]: ** Yg dibutuhkan** not appearing in bold

File: docs/schedule/reference.id.md [hunk 8]
```diff
@@ -353,20 +355,35 @@ Untuk contoh yang menunjukkan cara menentukan struktur tarif dengan fare_rules.t
...
+| `fare_media_type` |  enum           | ** Yg dibutuhkan** |  Jenis tarif media. Opsi yang valid adalah:<br/><br/>`0` - Tidak ada. Digunakan saat tidak ada media tarif yang terlibat dalam pembelian atau validasi produk tarif, seperti membayar tunai kepada pengemudi atau kondektur tanpa tiket fisik yang disediakan.<br/>`2` - Kartu transit fisik yang menyimpan tiket, pass, atau nilai uang.<br/>`3` - cEMV (Europay, Mastercard, dan Visa tanpa kontak) sebagai wadah token loop terbuka untuk tiket berbasis akun.<br/>`4` - Aplikasi seluler yang telah menyimpan kartu transit virtual, tiket, pass, atau nilai uang. |
```
Comment (josee-sabourin) [inline docs/schedule/reference.id.md]: ** Yg dibutuhkan** not appearing in bold

File: docs/schedule/reference.id.md [hunk 9]
```diff
@@ -353,20 +355,35 @@ Untuk contoh yang menunjukkan cara menentukan struktur tarif dengan fare_rules.t
...
-| `amount`            |  Jumlah mata uang | ** Yg dibutuhkan** |  Biaya produk tarif. Mungkin negatif untuk menunjukkan diskon transfer. Mungkin nol untuk mewakili produk tarif yang gratis. |
-| `currency`          |  Kode mata uang   | ** Yg dibutuhkan** |  Mata uang biaya produk tarif.                                                                                               |
+|  Nama Bidang        |  Jenis            |  Kehadiran         | Keterangan                                                                                                                                                                       |
+| ------------------- | ----------------- | ------------------ |----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `fare_product_id`   |  Identitas        | ** Yg dibutuhkan** | Mengidentifikasi produk tarif.                                                                                                                                                   |
```
Comment (josee-sabourin) [inline docs/schedule/reference.id.md]: ** Yg dibutuhkan** not appearing in bold

File: docs/schedule/reference.id.md [hunk 10]
```diff
@@ -353,20 +355,35 @@ Untuk contoh yang menunjukkan cara menentukan struktur tarif dengan fare_rules.t
...
+| `fare_product_name` |  Teks             |  Opsional          | Nama produk tarif yang ditampilkan kepada pengendara.                                                                                                                            |
+| `fare_media_id` | Referensi ID asing `fare_media.fare_media_id` | Opsional | Mengidentifikasi media tarif yang dapat digunakan untuk menggunakan produk tarif selama perjalanan. Ketika `fare_media_id` adalah kosong, dianggap sebagai media tarif tidak dikenal. |
+| `amount`            |  Jumlah mata uang | ** Yg dibutuhkan** | Biaya produk tarif. Mungkin negatif untuk menunjukkan diskon transfer. Mungkin nol untuk mewakili produk tarif yang gratis.                                                      |
```
Comment (josee-sabourin) [inline docs/schedule/reference.id.md]: ** Yg dibutuhkan** not appearing in bold

File: docs/schedule/reference.id.md [hunk 11]
```diff
@@ -353,20 +355,35 @@ Untuk contoh yang menunjukkan cara menentukan struktur tarif dengan fare_rules.t
...
+| `currency`          |  Kode mata uang   | ** Yg dibutuhkan** | Mata uang biaya produk tarif.                                                                                                                                                    |
```
Comment (josee-sabourin) [inline docs/schedule/reference.id.md]: ** Yg dibutuhkan** not appearing in bold

File: docs/schedule/reference.ko.md [hunk 12]
```diff
@@ -361,18 +363,33 @@ _예:_trip_id _그리고_stop_sequence _필드는 기본 키를 만듭니다._st
...
+
+파일: **선택 사항**
+
+기본 키(`fare_media_id`)
+
+요금 상품을 사용하는 데 사용할 수 있는 다양한 요금 매체를 설명합니다. 요금 매체는 요금 상품의 표시 및/또는 확인에 사용되는 물리적 또는 가상 소유자입니다.
+
+|  분야 명             |  유형    |  있음      |  설명                                                                                                                                                                                                                                                                                                                 |
+| ----------------- | ------ | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `fare_media_id`   |  고유 ID | ** 필수의** |  운임 매체를 식별합니다.                                                                                                                                                                                                                                                                                                      |
```
Comment (josee-sabourin) [inline docs/schedule/reference.ko.md]: ** 필수의** not appearing in bold

File: docs/schedule/reference.ko.md [hunk 13]
```diff
@@ -361,18 +363,33 @@ _예:_trip_id _그리고_stop_sequence _필드는 기본 키를 만듭니다._st
...
+| `fare_media_name` |  텍스트   |  선택 과목   |  운임 매체의 이름입니다.<br/><br/> 대중교통 카드인 요금 매체의 경우(`fare_media_type =2` ) 또는 모바일 앱(`fare_media_type =4` ),`fare_media_name` 포함되어야 하며 이를 제공하는 조직에서 사용하는 탑승자용 이름과 일치해야 합니다.                                                                                                                                                  |
+| `fare_media_type` |  열거    | ** 필수의** |  요금 매체의 유형입니다. 유효한 옵션은 다음과 같습니다.<br/><br/>`0` - 없음. 실물 티켓이 제공되지 않은 상태에서 기사나 차장에게 현금을 지불하는 것과 같이 운임 제품 구매 또는 확인과 관련된 운임 매체가 없을 때 사용됩니다.<br/>`2` - 티켓, 패스 또는 금전적 가치가 저장된 물리적 교통 카드.<br/>`3` - cEMV(비접촉식 Europay, Mastercard 및 Visa)는 계정 기반 발권을 위한 개방형 루프 토큰 컨테이너입니다.<br/>`4` - 가상 교통 카드, 티켓, 패스 또는 금전적 가치를 저장한 모바일 앱. |
```
Comment (josee-sabourin) [inline docs/schedule/reference.ko.md]: ** 필수의** not appearing in bold

File: docs/schedule/reference.ru.md [hunk 14]
```diff
@@ -355,18 +357,33 @@ GTFS V2 - это обновленный метод, позволяющий бо
...
+| `fare_media_type` | enum          | **Требуется** | Тип носителя информации о стоимости проезда. Допустимыми вариантами являются:<br/><br/>`0` - None.  Используется, когда в приобретении или проверке проездного продукта не участвует никакой носитель информации, например, оплата наличными водителю или кондуктору без предоставления физического билета.<br/>`2` - Физическая транзитная карта, на которой хранятся билеты, проездные билеты или денежные средства.<br/>`3` - cEMV (бесконтактная карта Europay, Mastercard и Visa) как контейнер для токенов открытого типа для продажи билетов на основе счета.<br/>`4` - Мобильное приложение, в котором хранятся виртуальные транзитные карты, билеты, проездные или денежная стоимость. |
+
 ### fare_products.txt
 
 Файл: **Необязательный**
 
-Первичный ключ`fare_product_id`)
+Первичный ключ (`fare_product_id`)
```
Comment (josee-sabourin) [inline docs/schedule/reference.ru.md]: missing fare_media_id

File: docs/schedule/reference.zh_TW.md [hunk 15]
```diff
@@ -116,7 +117,8 @@ search:
...
-| [fare_rules.txt](#fare_rulestxt)                   | 可选的  | 为行程应用票价的规则。                                                                                                                                                                        |
+| [fare_rules.txt](#fare_rulestxt)                   | 可选的  | 为行程应用票价的规则。                                                                                                                                                                                                                                                                          |
+| [fare_media.txt](#fare_mediatxt) | 可选的 | 描述可用於使用票價產品的票價媒體。<br/><br/>文件[fare_media.txt](#fare_mediatxt)描述了未在fare_attributes.txt和fare_rules.txt .因此， [fare_media.txt](#fare_mediatxt)的使用與文件完全分開[fare_attributes.txt](#fare_attributestxt)和[fare_rules.txt](#fare_rulestxt).                                                     |
```
Comment (josee-sabourin) [inline docs/schedule/reference.zh_TW.md]: missing hyperlinks for fare_attributes.txt and fare_rules.txt in first sentence

File: docs/schedule/reference.zh_TW.md [hunk 16]
```diff
@@ -356,19 +358,33 @@ search:
...
-首要的关键 （fare_product_id )
+首要的关键 （`fare_product_id`, `fare_media_id`)
 
 描述乘客可以购买的不同类型的车票或票价。
 
 | 字段名称                | 类型   | 在场      | 描述                                 |
 | ------------------- | ---- | ------- | ---------------------------------- |
 | `fare_product_id`   |  ID  | **必需的** | 标识票价产品。                            |
 | `fare_product_name` | 文本   | 可选的     | 向乘客显示的票价产品的名称。                     |
+| `fare_media_id` | 国外身份证参考 `fare_media.fare_media_id` |  標識可用於在旅行期間使用票價產品的票價媒體。當`fare_media_id`是空的，認為票價媒體是未知.|
```
Comment (josee-sabourin) [inline docs/schedule/reference.zh_TW.md]: error with the columns here

**Review by josee-sabourin — CHANGES_REQUESTED** [review]

**Review by josee-sabourin — APPROVED** [review]

**Pull request merged**